spec/plugins/data_table/server_side.js

```diff
@@ -290,15 +306,238 @@ describe("DataTable Plugin", function() {
 
     it("should default to GET when an AJAX method is not specified", function() {
       app.view.dataTable("AjaxMethodTestTable", {
-        rowClassName : "R",
-        serverSide : true,
+        rowClassName: "R",
+        serverSide: true
       });
 
-      table = new app.Views.AjaxMethodTestTable({ collection : collection });
+      table = new app.Views.AjaxMethodTestTable({collection: collection});
       table.render();
 
       expect(jasmine.Ajax.requests.mostRecent().method).toEqual("GET");
     });
+
+    it("should handle DT 1.10 param for sEcho (draw) for ajax response", function() {
+      table = new app.Views.T({ collection : collection });
+
+      var callbackSpy = jasmine.createSpy("callbackSpy");
```

Comment:
This block seems to be repeated throughout the some of the tests. If possible, can we move it out? However, it's only 4 lines, and dealing with the "this" context can get convoluted, so I can live with it repeating in the tests.